src/screens/People/People.js

```diff
@@ -436,11 +487,28 @@ class PeopleScreen extends React.Component<Props, State> {
             }
 
             {!inSearchMode && !sortedLocalContacts.length &&
-            <Wrapper center fullScreen style={{ paddingBottom: 100 }}>
-              <EmptyStateParagraph
-                title="Start making friends"
-                bodyText="Build your connection list by searching for someone"
-              />
+            <Wrapper fullScreen flex={1}>
+              <EmptyStateWrapper>
+                <EmptyStateParagraph
+                  title="Start making friends"
+                  bodyText="Build your connection list by searching for someone"
+                />
+              </EmptyStateWrapper>
+              {!!smartWalletFeatureEnabled &&
```

Comment:
Do we want to check Smart Wallet feature flag or Referrals?

Reply:
Updated